compose/ogmios/Dockerfile

```diff
@@ -0,0 +1,77 @@
+ARG CARDANO_NODE_VERSION=8.1.2
```

Comment:
:+1: We can specify any version at build time, so it's best to use the latest release here.